### PR TITLE
Add new police orgs to lookup

### PR DESCRIPTION
**Pull request by mpuckett159**
Adding:
 * Auburn, WA
 * Lakewood, WA
 * Olympia, WA

Police orgs to the look up tool.

### CODE REVIEW

File: db/sql/2_seattle.sql [hunk 1]
```diff
@@ -12,4 +12,4 @@ CREATE TABLE IF NOT EXISTS seattle_officers (
 );
 
 COPY seattle_officers (badge,full_name,title,unit,unit_description,first_name,middle_name,last_name,date)
-FROM '/seed/SPD_Rosters_Historical.csv' DELIMITER ',' CSV HEADER;
+FROM '/seed/Seattle-WA-Police-Department_Historical.csv' DELIMITER ',' CSV HEADER;
```
Comment (AetherUnbound) [inline db/sql/2_seattle.sql:15]: Good call on renaming these

**Review by AetherUnbound — APPROVED** [review]
LGTM, great work Marcus!!

File: db/sql/2_seattle.sql (hunk 1 above)
Comment (mpuckett159) [reply db/sql/2_seattle.sql:15]: Something we may have to look into down the line is updating the api routes which just use the city names atm, as well as the "ID" returned by the DepartmentMetadata object as it just uses a three character designator and that'll get confusing fast if we start adding more policing orgs.

Comment (AetherUnbound) [reply db/sql/2_seattle.sql:15]: Ah yea that's a really good point. Not urgent now but there's sure to be a lot of SPDs and TPDs out there

Comment (mpuckett159) [reply db/sql/2_seattle.sql:15]: I shall make a few issues for this.

**Pull request merged**